concent_api/gatekeeper/tests/test_gatekeeper_views.py

```diff
@@ -168,3 +198,26 @@ def test_download_should_return_401_if_wrong_token_cluster_address(self):
         self.assertIsInstance(response, JsonResponse)
         self.assertEqual(response.status_code, 401)
         self.assertIn('message', response.json().keys())
+
+    @freeze_time("2018-12-30 11:00:00")
+    def test_download_should_return_401_if_file_paths_are_not_unique(self):
+        self.upload_token.files.append(FileTransferToken.FileInfo())
+        self.upload_token.files[1]['path']      = 'blender/benchmark/test_task/scene-Helicopter-27-cycles.blend'
```

Comment:
You should `assert` that the file name in the existing `FileInfo` is the same as in your `FileInfo`. Otherwise the test is broken.

Reply:
Still no `assert` that the names are the same.

Reply:
I create new `FileInfo` object, so it doesn't matter.

Reply:
It does `assert` is also useful to indicate your assumptions. Adding it here shows clearly that you intended the names to be the same.